### DISCUSSION TITLE
Question: Can I install vips 8.6+ on Ubuntu 16.04 ?

### DISCUSSION THREAD

**Discussion by olimart** [libvips/libvips#2701] [Q&A]
When installing vips on Ubuntu 16.04 it installed libvips.42 so I'm assuming it's the only version available.
Is this correct? (or can I still install 8.6?)
How can I check the min. version required?

Thank you for your work.

**Comment by jcupitt** [comment]
Hello @olimart,

Sure, it should work. `42` is the ABI version, so everything since (I think?) 8.1 has been that. 

This dockerfile builds current libvips on 16.04, it might be helpful:

https://github.com/jcupitt/docker-builds/tree/master/php-vips-ubuntu-16.04
